colabs/raytune/W&B + RayTune.ipynb

```diff
@@ -6,7 +6,8 @@
       "name": "W&B-RayTune.ipynb",
```

Comment:
<p>Line 9: This comment is superfluous, I think, because you explain what the <code>Logger</code> does above. Instead, a comment like <code># WandbLogger uses tune.run's logger interface</code> seems like a better choice.</p><p>Line 10: Drop the newline here</p><p>Line 12: Make sure you consistently use either <code>"</code> or <code>'</code> for strings (here, <code>'api_key'</code> ).</p><p>Line 13: Drop the newline here</p><p>Line 16: Drop the newline here</p><p>Line 17: A longer comment would be better: <code># Hyperparameters are set by keyword arguments</code></p><p>Line 20: Watch the alignment of your parens there. Move the <code>}</code> up a line; perhaps move both. This is also part of PEP8.</p><p>Line 25: Why is this included? It's not used or explained.</p><p> </p><p> </p><p>There's another code cell below this one that you should delete.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>